ext/curl/multi.c

```diff
@@ -60,12 +60,18 @@ static inline php_curlm *curl_multi_from_obj(zend_object *obj) {
 PHP_FUNCTION(curl_multi_init)
 {
 	php_curlm *mh;
+	CURLM *multi;
 
 	ZEND_PARSE_PARAMETERS_NONE();
-
+	multi = curl_multi_init();
+	if(multi == NULL) {
+		// todo: should it throw or return false? (curl_init returns false)
+		php_error_docref(NULL, E_WARNING, "Could not initialize a new cURL multi handle");
```

Comment:
The stub says this function returns a `CurlMultiHandle`, so this can't return false. Instead, it should probably throw.

Reply:
changed it to throw :)